include/netdef.h

```diff
@@ -217,3 +227,5 @@ netplan_netdef_get_link_local_ipv6(const NetplanNetDefinition* netdef);
  */
 NETPLAN_PUBLIC ssize_t
 netplan_netdef_get_macaddress(const NetplanNetDefinition* netdef, char* out_buffer, size_t out_buffer_size);
+
+
```

Comment:
nitpick (non-blocking): Unnecessary newlines.

Reply:
I fixed this in https://github.com/canonical/netplan/pull/466/commits/692f9023432ea35cc5e36485e7c5dd12ecd61b96